extractorapp/src/main/java/org/georchestra/extractorapp/ws/extractor/ExtractorController.java

```diff
@@ -71,8 +73,29 @@
 
 	private ExtractionManager           extractionManager;
 
+	@Autowired
+	private GeorchestraConfiguration georConfig;
+
 	public void validateConfig() {
-		if(extractionManager==null) {
+	    if ((georConfig != null) && (georConfig.activated())) {
+	        LOG.info("geOrchestra datadir: reconfiguring bean ...");
+	        servletUrl = georConfig.getProperty("servletUrl");
+	        secureHost = georConfig.getProperty("secureHost");
+	        maxCoverageExtractionSize = Long.parseLong(georConfig.getProperty("maxCoverageExtractionSize"));
+	        remoteReproject = Boolean.parseBoolean(georConfig.getProperty("remoteReproject"));
+	        useCommandLineGDAL = Boolean.parseBoolean(georConfig.getProperty("useCommandLineGDAL"));
+	        extractionFolderPrefix = georConfig.getProperty("extractionFolderPrefix");
+	        String username = georConfig.getProperty("privileged_admin_name");
+            String password = georConfig.getProperty("privileged_admin_pass");
```

Comment:
The indentation of this file is completely messed up, I'll do a specific commit to reindent it completely separated from the previous comment fixes